tests/test_envs.py

```diff
@@ -23,21 +25,39 @@
     "seals/InitShiftTest-v0",
 ]
 
-ATARI_V5_ENVS: List[str] = [
-    "seals/" + env_name + "-v5" for env_name in seals.ATARI_ENV_NAMES
-]
-ATARI_NO_FRAMESKIP_ENVS: List[str] = [
-    "seals/" + env_name + "NoFrameskip-v4" for env_name in seals.ATARI_ENV_NAMES
+ATARI_ENVS: List[str] = [
+    seals._seals_name(gym_spec) for gym_spec in seals.GYM_ATARI_ENV_SPECS
 ]
 
-DETERMINISTIC_ENVS += ATARI_NO_FRAMESKIP_ENVS
+ATARI_V5_ENVS: List[str] = list(filter(lambda name: name.endswith("-v5"), ATARI_ENVS))
+ATARI_NO_FRAMESKIP_ENVS: List[str] = list(
+    filter(lambda name: name.endswith("-v4"), ATARI_ENVS),
+)
 
-ATARI_ENVS: List[str] = ATARI_V5_ENVS + ATARI_NO_FRAMESKIP_ENVS
+DETERMINISTIC_ENVS += ATARI_NO_FRAMESKIP_ENVS
 
 
 env = pytest.fixture(envs.make_env_fixture(skip_fn=pytest.skip))
 
 
+def test_some_atari_envs():
+    """Tests if we succeeded in finding any atari envs."""
+    assert len(seals.GYM_ATARI_ENV_SPECS) > 0
+
+
+def test_atari_space_invaders():
+    """Tests if there's an atari environment called space invaders."""
+    space_invader_environments = filter(
+        lambda name: "SpaceInvaders" in name,
+        ATARI_ENVS,
+    )
+    try:
```

Comment:
> This seems over-engineered. How expensive is it to just do `list(space_invader_environments)`? If cheap (I'd expect so), then doing that and `assert len(space_invader_environments) > 0` seems the way to go.

Probably cheap. It feels a bit sad to do it this way since really we only need to check that one element exists, rather than counting them all, but readability is probably more important.